lib/scan/detect_values.rb

```diff
@@ -56,6 +57,40 @@ def self.default_device
       raise "No simulators found".red unless Scan.device
     end
 
+    def self.default_device_tvos
+      config = Scan.config
+
+      if config[:device] # make sure it actually exists
+        device = config[:device].to_s.strip.tr('()', '') # Remove parenthesis
+
+        found = FastlaneCore::Simulator.all(requested_os_type: 'tvOS').find { |d| (d.name + " " + d.tvos_version).include? device }
```

Comment:
You can change this to be multi-line using the `do` and `end` block to make it easier to read :+1: